source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+To learn more, refer to our :manual:`support channels </support>`.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the C#/.NET driver,
+please open a case in our issue management tool, JIRA:
+
+* `Create an account and login <https://jira.mongodb.org>`_.
+* Navigate to `the .NET Driver project <https://jira.mongodb.org/browse/CSHARP>`_.
+* Click **Create Issue**. Please provide as much information as possible about the
```

Comment:
Suggestion: I think it works best to use an ordered list to represent steps that should be performed in order.
https://www.mongodb.com/docs/meta/style-guide/style/lists/#lists